honeybee_vtk/config.py

```diff
@@ -465,7 +465,13 @@ def load_config(json_path: str, model: Model, scene: Scene,
             'Not a valid json file.'
         )
     else:
+
+        # cwd = pathlib.Path.cwd()
+        jwd = pathlib.Path(json_path).absolute().parent
         for json_obj in config['data']:
+            if not pathlib.Path(json_obj['path']).is_dir():
```

Comment:
You probably want to check if the path is absolute instead of checking if it is a directory.

Reply:
I tried this and it does not work in all cases.